### DISCUSSION TITLE
dependabot[bot] included in contribution graph

### DISCUSSION THREAD

**Discussion by heaths** [community/community#17184] [Other Feature Feedback, Questions, & Ideas]
At the top of contributions e.g., https://github.com/cli/cli/graphs/contributors, it reads (emphasis mine),

> Contributions to trunk, excluding merge commits and *bot accounts*

However, @dependabot (the app, rather) is included: 

<img width="336" alt="image" src="https://user-images.githubusercontent.com/1532486/169886984-253750ac-6ef6-48d4-aa9a-a17f117f3d54.png">

**Comment by msimerson** [comment]
Agreed, bot activity should NOT, at least by default, show up in contribution displays. I've got `dependabot[bot]` and `greenkeeper[bot]` (defunct) garbage showing in my repos and I don't like it. I go out of my way to NOT merge commits from bots any longer, and turn off bot activity in development (dependabot.yml  `allow.dependency-type: "production"`). It would be better 

<img width="405" alt="Screen Shot 2022-06-01 at 2 29 01 PM" src="https://user-images.githubusercontent.com/261635/171505006-8dd6845d-9e5f-4052-9697-00c3192466b3.png">

<img width="265" alt="Screen Shot 2022-06-01 at 2 29 42 PM" src="https://user-images.githubusercontent.com/261635/171505077-50d6628c-31aa-4590-8904-4007db8af24c.png">

**Reply by msimerson** [reply]
<img width="165" alt="Screen Shot 2022-06-03 at 4 26 45 PM" src="https://user-images.githubusercontent.com/261635/171966536-ad4e396e-be7f-404a-8a29-c64fab87a7dc.png">

More ick. Half my contributors are bots and one of them is defunct.

**Comment by MarcusOtter** [comment]
Bumping this. I also filed a [bug report](https://support.github.com/contact/bug-report) - the bots are clearly supposed to be filtered out. It's unnecessary noise in the graph, and pushes actual contributors further down the list.

**Reply by kurtmckee** [reply]
@MarcusOtter Did you get a response?

**Reply by MarcusOtter** [reply]
I did, here's the conversation in August 2022 (omitted some parts for brevity):

GitHub Support person X
> They were always included in the count of contributors but their avatar was not displayed. This caused a lot of confusion because the count seemed wrong so we recently updated things so the avatars are displayed, to reflect the count accurately.

> If you like, I can pass your ticket to our Product Team to be treated as feedback that you don't like this change?

Me
> I still believe that the text "excluding merge commits and bot accounts" IS a bug, so that should be changed. Bots are included in the graph.

> But other than that, yes, I would like to send this as feedback. I understand that there was a design decision to include bots. My feedback is that I would like bot accounts excluded, or a way to filter which users to see. 

GitHub Support person Y
> Yes, I quite agree that the text saying bot accounts are excluded is a bug! I'm going to create an issue with the team to determine whether the text is accurate and the inclusion of the bot account in the chart is a regression, or whether including bot accounts is now deliberate and we need to update that text!

> I'm afraid our system is going to mark this ticket as "solved" now that the issue has been created with the engineering team, but that doesn't mean we've forgotten about it!

> We'll be back in touch when we have any updates!

I haven't heard anything since then and the text is still wrong

**Reply by kurtmckee** [reply]
Thanks for the update! I hope that they address this soon...
